database.js

```diff
@@ -0,0 +1,28 @@
+const { Client } = require("pg");
+require('dotenv').config()
+
+const client = new Client({
+  host: process.env.HOST,
+  port: process.env.PORT,
+  user: process.env.USERNAME,
+  password: process.env.PASSWORD,
+  database: process.env.DATABASE 
+});
+
+client.connect().then(() => {
+  createPgAccount("username", "password", "database");
+});
+
+const createPgAccount = async (username, password, database) => {
+  try{
+    const createDatabase = await client.query(`CREATE DATABASE ${database}`)
```

Comment:
Why do you need to assign the return the value to variables when it's not being used? 

It should just be `await client.query(CREATE DATABASE ${database}`)

Reply:
good point